.github/workflows/docs.yml

```
@@ -47,6 +47,9 @@ jobs:
           wayland: true
           xkb: true
 
+      - name: Install cargo-docs-rs
+        run: cargo install cargo-docs-rs
```

```suggestion
        run: cargo install cargo-docs-rs --version ^0.1.12
```

I like to specify the version. It prevents older versions and breaking changes from automatically being pulled in.